pdc/apps/package/tests.py

```diff
@@ -80,7 +80,7 @@ def test_empty_srpm_nevra_with_arch_is_not_src(self):
         self.assertEqual(0, models.RPM.objects.count())
 
 
-class RPMsListRESTTestCase(APITestCase):
+class RPMAPIRESTTestCase(APITestCase):
```

Comment:
Suggest involving `TestCaseWithChangeSetMixin`, for any changes(create/update), we can use its `assertNumChanges`